test-utils/testlib/src/standard_test_cases.rs

```diff
@@ -341,11 +342,11 @@ pub fn test_callback(node: RuntimeNode) {
 
     let mut state_update = node.client.read().unwrap().get_state_update();
     set(&mut state_update, key_for_callback(&callback_id), &callback);
-    let (root, transaction) = state_update.finalize();
+    let (transaction, root) = state_update.finalize();
```

Comment:
What is the reason to swap transaction and root?

Reply:
consistency, we were flipping them inside `finalize` before.